src/s2/s2region_term_indexer_test.cc

```diff
@@ -84,7 +84,7 @@ void TestRandomCaps(const S2RegionTermIndexer::Options& options,
     // random size.
     S2Cap cap;
     vector<string> terms;
-    if (query_type == QueryType::CAP) {
+    if (query_type == QueryType::POINT) {
```

Comment:
Test was correct but when was written
```
TestRandomCaps(options, QueryType::CAP);
```
it meant query points

and otherwise
```
TestRandomCaps(options, QueryType::Point);
```
meant caps

I think it's typo

Reply:
Yes, it looks like it was wrong. Thanks for fixing it.